test/schema.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    "domain": {
+        "type": "object",
+        "properties": {
+            "domain": {"type": "string"},
+            "owner": {"type": "object"},
+            "prevalence": {"type": "number"},
+            "fingerprinting": {"type": "integer", "minimum": 0, "maximum": 3},
+            "cookies": {"type": "number"},
+            "categories": {"type": "array"},
+            "default": {"type": "string"},
```

Comment:
can we do an enum with "block"/"ignore"?

Reply:
Removed 'default', not needed in here.